asset-tracking-location-replaying/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "asset-tracking-location-replaying",
+  "version": "1.0.0",
+  "description": "CLI tool that replays location updates",
+  "main": "index.ts",
+  "scripts": {
+    "start": "./node_modules/ts-node/dist/bin.js src/index.ts"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "ably": "^1.2.4",
+    "commander": "^7.0.0",
+    "death": "^1.1.0",
+    "ts-node": "^10.0.0",
+    "typescript": "^4.3.2"
+  },
+  "devDependencies": {
+    "@types/node": "^15.12.2"
+  }
+}
```

Comment:
Did you consider adding a `bin` property? see the `asset-tracking-debugger` tool for an example. If you add a bin property with a name like `asset-tracking-location-replayer` then people will be able to run `npm link` or `npm i -g <package-name>` and add it to their global path which might be pretty useful.

Reply:
Good suggestion :wink: https://github.com/ably/ably-asset-tracking-common/pull/25/commits/c70c81fa34eabb8c240cf71af6892ed4f11163bd